eclipse-plugin/service/cc.codechecker.service.api/src/main/java/cc/codechecker/api/runtime/CodeCheckEnvironmentChecker.java

```diff
@@ -45,8 +46,13 @@ public CodeCheckEnvironmentChecker(Optional<String> pythonEnvironment,
 
         Optional<String> ccEnvOutput = she.quickReturnOutput(codeCheckerCommand + " check -w " +
                 workspaceName + " -n dummy -b env | grep =");
-
-        if (!ccEnvOutput.isPresent()) {
+        double test = 2;
+        while(!ccEnvOutput.isPresent() && test <= 4) {
+        	ccEnvOutput = she.quickReturnOutput(codeCheckerCommand + " check -w " +
+                    workspaceName + " -n dummy -b env | grep =", Math.pow( 2.0 , test ) * 1000);
```

Comment:
You should lower this time as it can result in a high latency on eclipse startup / slow PCs. Or at least make the jobs cancellable, so the IDE can quickly kill them when the configuration changes.